topics/chrome/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: google-chrome
+related: firefox,safari,edge,opera,chromium
+created_by: Google
+display_name: Chrome
+github_url: https://github.com/googlechrome
+logo: chrome.png
+released: September 02, 2008
```

Comment:
If a test doesn't fail because of it, can you change this to `2` instead of `02`?